libgalois/include/katana/analytics/BfsSsspImplementationBase.h

```diff
@@ -38,7 +38,7 @@ struct BfsSsspImplementationBase {
   constexpr static const bool kTrackWork = false;
 
   constexpr static const Dist kDistanceInfinity =
-      std::numeric_limits<Dist>::max() / 4;
+      std::numeric_limits<Dist>::max();
```

Comment:
The same value of infinity should be used for bfs as well, right?

Reply:
It shouldn't matter. But I'll change BFS too. The same weird `max/4` pattern appears in other applications in both the CPU and distributed lonestar suites. I have not tried to fix them in general because I don't know why this number was chosen.

Reply:
Okay. Just use the same value for shared-memory bfs and sssp and we can fix others when we encounter them.

Reply:
So I had to roll back all the `/4` changes because it caused the tests to fail because the check data contains this constant and I didn't want to to make such a tiny change to the inputs on short notice. @witchel is waiting on this PR. I'll create a new PR to fix things like this in a bit. Is that OK?